ios/Connectors/PinConnector.swift

```diff
@@ -50,13 +50,13 @@ enum PinNotification : String {
 }
 
 // Pin actions from RN Bridge
-enum PinAction : NSString {
+enum PinAction : String {
     case provide = "provide",
```

Comment:
I haven't noticed it previously but when enum is type of String its cases do not need to have string values for keys (if values are equally the same as keys), which means:
```
enum PinAction: String {
  case provide
  case cancel
}```